tests/gcs/test_gcs_upload.py

```diff
@@ -0,0 +1,27 @@
+from google.cloud import storage
+
+
+
+def upload_blob(bucket_name, source_file_name, destination_blob_name):
+    """Uploads a file to the bucket."""
+    # The ID of your GCS bucket
+    # bucket_name = "your-bucket-name"
+    # The path to your file to upload
+    # source_file_name = "local/path/to/file"
+    # The ID of your GCS object
+    # destination_blob_name = "storage-object-name"
+
+    storage_client = storage.Client()
+    bucket = storage_client.bucket(bucket_name)
+    blob = bucket.blob(destination_blob_name)
+
+    blob.upload_from_filename(source_file_name)
+    
+    print(
+        f"File {source_file_name} uploaded to {destination_blob_name}."
+    )
+
+
+if __name__ == "__main__":
+    BUCKET='movies-wagon'
+    upload_blob(BUCKET, 'test.csv', 'data/test.csv')
```

Comment:
* on a un .env 
* il faudrait un chemin relatif pour test.dummy

Reply:
Good catch 🧐
- Let's use direnv to have acces to the `GCP_BUCKET` var in .env
- On it, will make the path to test.dummy relative